zbsprimarystorage/ansible/zbsp.py

```
@@ -114,6 +114,11 @@
 set_selinux("state=disabled", host_post_info)
 
 
+# name: make sure MDS has access to storage resources
+command = "[ -f /usr/local/bin/zbsadm ] && /usr/local/bin/zbsadm client deploy --host %s -p %s" % (host_post_info.host, host_post_info.remote_pass)
+run_remote_command(command, host_post_info)
```

Ruff (0.11.9)

`run_remote_command` may be undefined, or defined from star imports

(F405)